.github/workflows/release.yml

```diff
@@ -31,27 +32,28 @@ jobs:
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
         with:
           fetch-depth: 0
+          persist-credentials: false
+
       - name: Batch changes
         uses: miniscruff/changie-action@b6d52c80deb236a5b548f8774cd5a18b87da9e9a # v1.0.1
         with:
           version: latest
           args: batch ${{ needs.changelog-version.outputs.version }}
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Merge changes
         uses: miniscruff/changie-action@b6d52c80deb236a5b548f8774cd5a18b87da9e9a # v1.0.1
         with:
           version: latest
           args: merge
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Git push changelog
         run: |
           git config --global user.name "${{ env.CI_COMMIT_AUTHOR }}"
           git config --global user.email "${{ env.CI_COMMIT_EMAIL }}"
           git add .
-          git commit -a -m "Update changelog"
-          git push
+          git commit -a -m "[skip ci] Update changelog"
```

Comment:
What's the `[skip ci]` bit here? I think that may have some meaning in other repositories, but I'm not aware of anything for ours.

Reply:
So this is a supported command (among others documented [here](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs)) that will skip any `push` or `pull_request` triggered github workflows.

I don't think many of our repo's have `push` workflows, and a lot filter out files like `CHANGELOG.md`, so this is more preventative then anything.

Reply:
Okay I'm with you up until:

> more preventative then anything

Preventative of what though? I'm not sure it is costing us anything to have CI potentially trigger on the commit itself, other than potentially showing the commit with potentially unrelated failed checks. You may notice one of my themes with "non defaults" is asking why and/or commenting them because over time that origin knowledge gets lost. Should I be concerned about potentially removing that syntax from the commit message in the future? Maybe new branch protection rules allow you to prevent this exact commit message behavior and this workflow becomes broken. 😄

Reply:
Ah okay that makes sense, I'm on-board 👍🏻 - reverting back to original message